src/Orleans.Core/Configuration/Options/VersioningOptions.cs

```diff
@@ -1,16 +1,25 @@
 
+using System.Collections.Generic;
 using Microsoft.Extensions.Options;
 using Orleans.Versions.Compatibility;
 using Orleans.Versions.Selector;
-using System.Collections.Generic;
 
 namespace Orleans.Hosting
 {
+    /// <summary>
+    /// Versioning options govern grain implementation selection in heterogeneous deployments.
+    /// </summary>
     public class VersioningOptions
```

Comment:
We should probably rename this GrainVersioningOptions since it's not relevant to model versioning or silo/client versioning